### PR TITLE
Re-publish existing openwhisk-client-js release to fix CI failure on last publish

### PR DESCRIPTION
**Pull request by jthomas**
Once voting has closed on previous RC, CI setup had changed which meant I was unable to publish to the official release directory. See build failure here after tagging release: 

https://travis-ci.org/apache/incubator-openwhisk-release/builds/515296930

This PR will re-publish existing artifacts from repo with updated CI build system allowing me to tag the release again for publishing to apache release site.

### CODE REVIEW

File: tools/config.json [hunk 1]
```diff
@@ -4,16 +4,15 @@
...
-    "pre_release_version": "rc2"
+    "version": "3.19.0-incubating",
+    "pre_release_version": "rc1"
   },
   "RepoList": [
-      "openwhisk-composer"
+    "openwhisk-client-js"
   ],
-  "openwhisk_composer": {
-    "name": "OpenWhisk Composer",
```
Comment (rabbah) [inline tools/config.json]: the config file should have the `name` property as on line 14.

**Comment by rabbah** [comment]
This is good evidence to suggest limiting the use of Travis _on_ the upstream fork for creating releases since it leads to collisions and instead release managers should use their own forks of this repo.

**Comment by dgrove-oss** [comment]
@jthomas -- if you have existing artifacts that you just want to move from dev to dist it might be easier to just do the svn commands manually.

**Comment by houshengbo** [comment]
@rabbah In a long run, it turns out to be each committer or release manager operates his/her own fork of this release repository to manage the release of the projects in need. Is this also what you suggest and foresee? 

Manual mode is always an option, and a bottom line option, for the release manager, if there is any type of conflict. However, the conflict only happens in the config file, that we can cope anyhow.

File: tools/config.json (hunk 1 above)
Comment (houshengbo) [reply tools/config.json]: It is good to have, but I do think it is mandatory for the release build and push in Travis.

**Comment by jthomas** [comment]
@dgrove-oss Good point - I probably should have thought of that! 

I do think (in general) this process works well. This example was a fairly unique edge case where the CI setup changed during the voting process.

**Pull request merged**